builtin/common/misc_helpers.lua

```diff
@@ -344,17 +344,11 @@ if INIT == "game" then
 --Wrapper for rotate_and_place() to check for sneak and assume Creative mode
 --implies infinite stacks when performing a 6d rotation.
 --------------------------------------------------------------------------------
-	local creative_mode_cache = core.settings:get_bool("creative_mode")
-	local function is_creative(name)
-		return creative_mode_cache or
-				core.check_player_privs(name, {creative = true})
-	end
```

Comment:
Any reason to remove privilege check and force me to override this function to return creative for only 1 player (me)  on my survival server?

Reply:
Is the `creative` privilege documented anywhere? If not, the engine should not have code to support it.

Reply:
Then let's document it. This is really a trivial issue.

Reply:
The engine doesn't have a creative priv. That's in the creative mod. The creative mod can just override this to add the creative priv

Reply:
@rubenwardy, so, the engine supports per-player immortal (starting 5.1), per-player sky, per-player time, per-player FOV, etc.  As well as two dozen vaunting privileges.
But per-player creative should strictly add each subgame by each developer?